src/domain/app/header/userDropdown/UserDropdown.tsx

```diff
@@ -32,11 +32,15 @@ const UserDropdown: React.FunctionComponent = () => {
         {
           label: t('authentication.logout.text'),
           onClick: () => {
+            // Clear user form data
             dispatch(resetFormValues());
-            // Clear user form data from redux store
-            persistor.flush();
+            // Clear profile (fetched from API)
+            dispatch(clearProfile());
+            // Clear backend auth data
+            dispatch(fetchTokenError());
```

Comment:
Do you really need to dispatch an error or do you really want to just clear the state?

Reply:
Uhm, this is maybe a badly named action :D

Reply:
It looks like the action is used for just that intent (token fetch error), I think it would be best to create a new action + reducer for resetting the backend auth state.

Reply:
Better now?